### ISSUE TITLE
Replace formula images with MathJax

### ISSUE THREAD

**Issue by tobyhodges** [carpentries-lab/metagenomics-analysis#31]
Mathematical formulae are displayed as images in [_Diversity Tackled with R_](https://carpentries-incubator.github.io/metagenomics/07-Diversity-tackled-with-R/index.html#%CE%B1-diversity) without alternative text. These images are inaccessible for anyone visiting the lesson using a screen reader. 

To make the lesson more accessible, and easier to maintain, the lesson template allows you to replace these images with [MathJax](https://www.mathjax.org/) elements. Follow the steps below to replace these images.

1. Add the line `math: yes` to the bottom of your `_config.yml` file.
2. In the episode file, replace the images of formulae with the LaTeX equivalent, wrapped in `$` signs, e.g.

```
$$E(y) = \beta_0 + \beta_1 \times x_1 + \beta_2 \times x_2.$$
```

For an example of what I describe above, you can browse the [`_config.yml`](https://github.com/carpentries-incubator/multiple-linear-regression-public-health/blob/main/_config.yml#L102) and [episode page source](https://github.com/carpentries-incubator/multiple-linear-regression-public-health/blob/main/_episodes/01-contAndCat.md) of [the _Multiple Linear Regression for Public Health_ lesson](https://carpentries-incubator.github.io/multiple-linear-regression-public-health/) in the Incubator.

[See the Lesson Example for further documentation of this feature of the lesson template](https://carpentries.github.io/lesson-example/04-formatting/index.html#adding-formatted-equations).

**Comment by nselem** [comment]
dear toby, we have tried. We add `math: yes` to _`config.yml`  and `math: true` at the yaml of the file as shown in the example, but latex does not properly display at the lesson. Any ideas?

**Comment by tobyhodges** [comment]
Hi @nselem it looks like it is working to me?

Here is a screenshot from https://carpentries-incubator.github.io/metagenomics/07-Diversity-tackled-with-R/index.html

![inline math rendering in the page](https://user-images.githubusercontent.com/9694524/168111867-1abe7f56-d18f-48d8-8d3d-81b0fc24cf63.png)

and here is one from my local build, after I added the example block below

```
$$E(y) = \beta_0 + \beta_1 \times x_1 + \beta_2 \times x_2.$$
```

![block of MathJax](https://user-images.githubusercontent.com/9694524/168112354-61f60473-ec7b-4e3b-ad6a-5b1e60c3cd1e.png)

**Comment by nselem** [comment]
No, It does not look like yours. 
![image](https://user-images.githubusercontent.com/6643162/168122415-123f2f0d-1601-49f8-8a04-b94efaba227f.png)

**Comment by tobyhodges** [comment]
I think the difference in styling is down to us using different browsers to view the lesson (I am on Firefox 100.0 + MacOS 11.6.5), but we can't see the `$` symbols so _something_ is working in your screenshot. Could you take a look in a different web browser and see if it looks any different there? @zkamvar any other suggestions about what might be happening here?

**Comment by nselem** [comment]
I am using Chrome, on edge looks like in Chrome

**Comment by tobyhodges** [comment]
perhaps something to do with these error messages I get in the browser console @zkamvar?

```
downloadable font: download failed (font-family: "KaTeX_Main" style:normal weight:400 stretch:100 src index:0): status=2147746065 source: https://carpentries-incubator.github.io/metagenomics/assets/css/fonts/KaTeX_Main-Regular.woff2
downloadable font: download failed (font-family: "KaTeX_Main" style:normal weight:400 stretch:100 src index:1): status=2147746065 source: https://carpentries-incubator.github.io/metagenomics/assets/css/fonts/KaTeX_Main-Regular.woff
downloadable font: download failed (font-family: "KaTeX_Main" style:normal weight:400 stretch:100 src index:2): status=2147746065 source: https://carpentries-incubator.github.io/metagenomics/assets/css/fonts/KaTeX_Main-Regular.ttf
GEThttps://carpentries-incubator.github.io/metagenomics/assets/css/fonts/KaTeX_Main-Regular.woff2
[HTTP/2 404 Not Found 28ms]

downloadable font: download failed (font-family: "KaTeX_Main" style:normal weight:400 stretch:100 src index:0): status=2147746065 source: https://carpentries-incubator.github.io/metagenomics/assets/css/fonts/KaTeX_Main-Regular.woff2
GEThttps://carpentries-incubator.github.io/metagenomics/assets/css/fonts/KaTeX_Main-Regular.woff
[HTTP/2 404 Not Found 24ms]

downloadable font: download failed (font-family: "KaTeX_Main" style:normal weight:400 stretch:100 src index:1): status=2147746065 source: https://carpentries-incubator.github.io/metagenomics/assets/css/fonts/KaTeX_Main-Regular.woff
GEThttps://carpentries-incubator.github.io/metagenomics/assets/css/fonts/KaTeX_Main-Regular.ttf
[HTTP/2 404 Not Found 27ms]

downloadable font: download failed (font-family: "KaTeX_Main" style:normal weight:400 stretch:100 src index:2): status=2147746065 source: https://carpentries-incubator.github.io/metagenomics/assets/css/fonts/KaTeX_Main-Regular.ttf
```

**Comment by zkamvar** [comment]
I NOW UNDERSTAND~~am so confused~~ as to why this is happening for this particular lesson.

1. support for math fonts was added to the styles repo in https://github.com/carpentries/styles/pull/573 and propagated to the carpentries-theme repo sometime later by @fmichonneau 
2. A draft PR was created a week later to add in the fonts that were erroring, but the author never marked it as ready to merge, so it got stuck in limbo: https://github.com/carpentries/styles/pull/592

The reason I ~~am confused~~now understand is because the setup for [Multiple Linear Regression Public Health](https://carpentries-incubator.github.io/multiple-linear-regression-public-health/01-contAndCat/index.html) is _slightly different_ ~~is pretty much the same (`math: true` in both config and episode) and yet the math fonts there _actually_ render 😩.~~

UPDATE: it turns out that the author of Multiple Linear Regression Public Health added MathJax support manually: https://github.com/carpentries-incubator/multiple-linear-regression-public-health/commit/5f94605b6d65c91419441326c00a8eb119b249b4#diff-b17352b6dc874d1a1f9a927f82a383a94884bc593087f1636d05118e250852db

So, the quickest way to fix this is to add `_includes/javascript.html` into this repository to allow math, but the most stable path going forward is to merge https://github.com/carpentries/styles/pull/592 and then have @fmichonneau propagate the changes to carpentries-theme

**Comment by nselem** [comment]
Hey thank you very much to both of you. We have manually uploaded the HTML file and the visualization is working now. @tobyhodges  could you please add as collaborators to karinaeg and RafaPe  ? They will help us with these changes.

**Comment by tobyhodges** [comment]
@nselem I have added @rafape and @karinaeg ~~but I cannot find a GitHub user named `karinaeg` - can you check and confirm the username please?~~

**Comment by RafaPe** [comment]
I just finished changing the math images on the "Diversity Tackled with R" episode with MathJax elements. I would really appreciate if @nselem and @tobyhodges could check it and let me know if the episode is now as expected so we can close the issue.

**Comment by nselem** [comment]
Thanks, @RafaPe Latex looks fine now, by the way, @tobyhodges, I don't know why this episode does not show the "improve this page button" any ideas?

**Comment by tobyhodges** [comment]
@RafaPe it looks great to me. Thank you for taking the time to make this change: it results in an episode that, to me, looks just like the old version, but your changes have made the lesson much more accessible for screen reader users, and I believe this approach is more maintainable too 😄 

@nselem: I do not understand exactly what the issue is, but you can fix it by removing the `source: md` line from the YAML header of the episode file. It is only necessary to specify `source` when your lesson is being built from RMarkdown files stored in a separate branch from `gh-pages`.

**Comment by nselem** [comment]
Dear @tobyhodges @fpsom and @PaulaNietoG 
We have considered all your valuable suggestions and worked on them as individual issues in the Metagenomics lesson. We thank you very much for your detailed observations that have helped us to improve the material significantly. 
A summary of the main changes is the following list: 
- The [setup](https://carpentries-incubator.github.io/metagenomics-workshop/setup.html) page was cleaned and improved. Now it explains how to use the different options for obtaining the software; the Amazon Web Services remote machine and the local installation of the programs using a Conda environment in the three main operating systems. For AWS, a new instance version has been created, and [instructions ](https://carpentries-incubator.github.io/metagenomics-workshop/AMI-setup/index.html) have been updated. All the software is now included in the conda environment for the local installation.
- A copy of all the files needed while running the lesson was added as a hidden directory and uploaded in [Zenodo](https://zenodo.org/record/7010950#.Y8uE43bMLrc). This backup can be used if a learner cannot generate some intermediate files. The Amazon Image was updated with this new data folder and the Conda environment. All missing files, such as the adapters file, are now included in Zenodo. This change is explained in the [Instructor Notes](https://carpentries-incubator.github.io/metagenomics-workshop/guide/index.html).
- To diminish the cognitive load in the [Data Processing and Visualization for Metagenomics](https://carpentries-incubator.github.io/metagenomics/), the content was rearranged and split into four instead of three episodes r-based episodes. Now, in the first one the OTU table is explained, the other talks about diversity, and finally, in the last one abundance graphics are produced. Some of the content was moved to the Introduction to R lesson, so the taxonomy episodes are a little less dense.
- To diminish the cognitive load in the [Introduction to the Command Line](https://carpentries-incubator.github.io/shell-metagenomics/), the `scp` command was left as optional. We use the export function in the r studio interface as an alternative. In our experience in the workshop, this was a difficult part for learners, nevertheless, it is valuable, so we left it there so they can come back and keep learning if they want. The rest of the material, in our experience, works well even for newcomers in the given amount of time.
- The exercises and discussions were improved, and solutions were added to the exercises that did not have them. New conceptual discussions were added to several episodes.
- Figures were complemented with missing alternative text. 
- A general review of the typos and writing was made in all the episodes.
Now I will point to the issues that address punctual observations that need to be improved. 
- New and cleaned setup: [issue#62](https://github.com/carpentries-incubator/metagenomics/issues/62)
- Typos [issue#61](https://github.com/carpentries-incubator/metagenomics/issues/61)
- Adding conceptual questions [issue#60](https://github.com/carpentries-incubator/metagenomics/issues/60)
- Provide all required files[issue#59](https://github.com/carpentries-incubator/metagenomics/issues/59)
- Adapter file and multiline command in [Trimming and Filtering](https://carpentries-incubator.github.io/metagenomics/03-trimming-filtering/index.html) episode [issue#58](https://github.com/carpentries-incubator/metagenomics/issues/58)
- Exercise 2 in [Assessing Read Quality](https://carpentries-incubator.github.io/metagenomics/02-assessing-read-quality/index.html) episode. [issue#57](https://github.com/carpentries-incubator/metagenomics/issues/57)
- Cognitive load in [Data processing and visualization for metagenomics](https://carpentries-incubator.github.io/metagenomics/index.html) and splitting into smaller episodes.  [issue#56](https://github.com/carpentries-incubator/metagenomics/issues/56)
- R datatypes [issue#54](https://github.com/carpentries-incubator/metagenomics/issues/54)
- Some Spanish names. [issue#53](https://github.com/carpentries-incubator/metagenomics/issues/53)
- Misleading name for R lesson [issue#52](https://github.com/carpentries-incubator/metagenomics/issues/52)
- Improve color palette for color blindness. [issue#51](https://github.com/carpentries-incubator/metagenomics/issues/51)
- Repeated Learning Objectives [issue#50](https://github.com/carpentries-incubator/metagenomics/issues/50)
- Intro to the command line. [issue49](https://github.com/carpentries-incubator/metagenomics/issues/49) and simplified version of [Writing Scripts and Working with Data](https://carpentries-incubator.github.io/shell-metagenomics/05-writing-scripts/index.html)
- Improve [Writing Scripts and Working with Data](https://carpentries-incubator.github.io/shell-metagenomics/05-writing-scripts/index.html) episode. [issue#48](https://github.com/carpentries-incubator/metagenomics/issues/48)
- Improve  [Redirection](https://carpentries-incubator.github.io/shell-metagenomics/04-redirection/index.html) episode. [issue#47](https://github.com/carpentries-incubator/metagenomics/issues/47)
- [Working with Files and Directories](https://carpentries-incubator.github.io/shell-metagenomics/03-working-with-files/index.html) episode  [issue#46](https://github.com/carpentries-incubator/metagenomics/issues/46)
- [Introducing the Shell](https://carpentries-incubator.github.io/shell-metagenomics/01-introduction/index.html) episode  [issue#45](https://github.com/carpentries-incubator/metagenomics/issues/45)
- Alternative text of images [#44](https://github.com/carpentries-incubator/metagenomics/issues/44)
- Improve discussions in Project Organization and Management lesson [issue#43](https://github.com/carpentries-incubator/metagenomics/issues/43)
- Image transcription in Examining Data on the NCBI SRA Database [issue#42](https://github.com/carpentries-incubator/metagenomics/issues/42)
- Update the zenodo link in Examining Data on the NCBI SRA Database [issue#41](https://github.com/carpentries-incubator/metagenomics/issues/41)
- Improve Structuring data in spreadsheets in Data Tidiness [issue#40](https://github.com/carpentries-incubator/metagenomics/issues/40)
- Improve Data about the experiment in Data Tidiness [issue#39](https://github.com/carpentries-incubator/metagenomics/issues/39)
- Add confidentiality in Data Tidiness [issue#38](https://github.com/carpentries-incubator/metagenomics/issues/38)
- Improve Examining Data on the NCBI SRA Database episode [#37](https://github.com/carpentries-incubator/metagenomics/issues/37)
- Workshop prerequisites [issue#36](https://github.com/carpentries-incubator/metagenomics/issues/36)
- Develop Instructor Notes [issue#32](https://github.com/carpentries-incubator/metagenomics/issues/32)
